tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py

```diff
@@ -31,12 +31,20 @@ def test_click_run_stopwatch_then_reset(self):
 
         first_time = stopwatch_view.current_time
 
+        stopwatch_view.tap_resume()
+
         time.sleep(0.2)
 
         self.assertNotEqual(stopwatch_view.current_time, '00:00.00')
 
         self.assertNotEqual(first_time, stopwatch_view.current_time)
 
+        stopwatch_view.tap_pause()
+
+        time.sleep(0.2)
```

Comment:
I wonder what this sleep is. We should be able tap reset right after, shouldn't we?